modules/third-party/com-sun-el-javax-el/patches/LPS-141588.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/com/sun/el/parser/ELParser.java b/com/sun/el/parser/ELParser.java
```

Comment:
@dacousalr Please provide more details about the patched changes in the commit message. Eric told me there will be two different fixes for memory leaking, I suppose you put them together. Please try to separate it so that we are clear why we made this change, if there is any trouble and you can not split them, please add the details about the fix to the commit message so that we can trace back later.